src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -335,6 +339,17 @@ export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
                     title="OncoKB Matched Trials General Feedback"
                     userEmailAddress={AppConfig.serverConfig.user_email_address}
                 />
+                {   this.selectedTrialFeedbackFormData &&
+                    <TrialMatchFeedback
+                        show={!!this.selectedTrialFeedbackFormData}
+                        onHide={() => !!(this.selectedTrialFeedbackFormData=undefined)}
```

Comment:
onHide also modifies state so should be action.  What I would do is change openFeedbackForm to openCloseFeedbackForm and have it accept undefined if it's close

Reply:
Updated